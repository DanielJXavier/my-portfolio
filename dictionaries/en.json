{
  "global": {
    "author": {
      "headline": "Senior Full-Stack Developer | Front-End focused | React | Next.js | TypeScript | Node.js | JavaScript | Leadership"
    },
    "copyright": "All rights reserved.",
    "menu": {
      "experience": "Experience",
      "education": "Education",
      "skills": "Skills",
      "recommendations": "Recommendations"
    }
  },
  "home": {
    "greeting": "Hey, I'm",
    "summary": [
      "With more than 10 years of experience in web development and expertise in TypeScript, JavaScript, React, and Node.js, I have consistently delivered elegant solutions to complex challenges. My proficiency extends beyond technical skills, as I have successfully led teams on complex projects with relevant results for the company.",
      "As an experienced Front-End developer and leader, I have a proven track record of designing intuitive user interfaces and robust backend systems, ensuring high performance, scalability, and security. My leadership style is rooted in collaboration and continuous improvement, driving my teams to achieve excellence.",
      "Now, I am focused back on the hands-on approach, designing and developing robust and scalable solutions with the best practices contributing actively to the company's goals achieving great results together!"
    ],
    "imageAltText": "Photo of the author"
  },
  "experience": {
    "title": "Experience",
    "menu": {
      "simple": "simple",
      "detailed": "detailed"
    },
    "accomplishmentsTitle": "Main accomplishments",
    "biggestChallengeTitle": "Biggest challenge",
    "items": {
      "danielxavier-1": {
        "role": "Senior Full-Stack Developer",
        "description": "I am a Senior Full-Stack developer.",
        "accomplishments": [
          "High-performance and fully accessible professional website using React, Next.js, and Tailwind CSS;",
          "Template-based app for printing medical reports with React, and Next.js;",
          "Ticket booking API with Nest.js, and Kong;"
        ],
        "biggestChallenge": "To quickly conceptualize and release projects from scratch and with few defined requirements."
      },
      "olx-4": {
        "role": "Software Engineering Manager",
        "description": "I was the leader of the \"Ads Seller Experience\" team.",
        "accomplishments": [
          "I led a team of 9 developers, engaging them even in a crisis scenario;",
          "Delivered the new Ad Insertion journey in stages, giving users a fluid and easier experience to sell items in the classified platform;",
          "I mentored the team members through their performance cycles with clear and achievable goals for the next year;",
          "I led team ceremonies with agile practices to ensure a streamlined workflow improving task throughput in 50%;",
          "Worked closely with the Product Manager to mitigate risks and make effective product decisions;"
        ],
        "biggestChallenge": "To deal with pressure from leadership to do major deliveries in the short term."
      },
      "olx-3": {
        "role": "Software Engineering Manager",
        "description": "I was the leader of the \"Search & Recommendation\" team¹ and the \"Design System\" chapter².",
        "accomplishments": [
          "I led a team of 8 developers, leveraging the strengths of each of them;¹",
          "Deliver new LTR ranking algorithm that brings 10% more conversion;¹",
          "I structured the team into 3 independent squads to support the strategic plan for next year;¹",
          "I indirectly led all developers (9 devs) of the chapter, keeping the focus on the most valuable deliveries;²",
          "Launched the Design System v3 has been launched, bringing 30% more performance to all pages on the platform;²"
        ],
        "biggestChallenge": "To balance the substantial responsibilities of both the team¹ and the chapter² simultaneously."
      },
      "olx-2": {
        "role": "Software Engineering Manager",
        "description": "I was the leader of the 'Design System' team.",
        "accomplishments": [
          "I led a team of 9 developers, influencing technical decisions;",
          "Partnering leadership with the Design Manager to make optimal product decisions and define a product roadmap;",
          "Delivered the Design System roadmap, including a brand new platform home page, created in less than 48 hours;",
          "I ensured the continuous evolution of the Design System and its adoption by other teams;",
          "I created processes for contributing to the Design System project, bringing 10 new external contributors;"
        ],
        "biggestChallenge": "To assume multiple roles simultaneously, including Engineering Manager, Product Manager, Data Analyst, among others."
      },
      "olx-1": {
        "role": "Software Engineering Manager",
        "description": "I was the leader of the \"Advertising\" team.",
        "accomplishments": [
          "Led a team of 7 developers, serving as a key reference;",
          "Delivered some Advertising improvements for the platform that brought R$80 million in revenue;",
          "Recruited new professionals to increase productive capacity by 300%;",
          "Partnered actively with the Product Manager to prioritize and develop optimal initiatives;",
          "Proposed a new structure for the Advertising area to enhance results;"
        ],
        "biggestChallenge": "To manage high expectations related to Advertising revenue results."
      },
      "meaning-1": {
        "role": "CTO & Partner",
        "description": "I held the position of CTO and co-founder at the startup.",
        "accomplishments": [
          "Crafted the startup's technical vision and product strategy;",
          "Architected the solution from the ground up and delivered it to 4 new clients;",
          "Refined the product's technical specifications;",
          "Developed a Proof of Concept project;"
        ],
        "biggestChallenge": "To convert a highly \"human-centric\" project, rooted in personal consultancy, into a systematic approach."
      },
      "softplan-3": {
        "role": "Development Coordinator",
        "description": "I was the leader of the \"Estônia\" team.",
        "accomplishments": [
          "Led directly a multidisciplinary software development team of 28 people;",
          "Planned and executed deliveries for the most 2 critical modules of the \"SAJ\" system;",
          "Structured and continuously enhanced the team's workflow using agile software development methodologies;",
          "Created formulas and metrics to accurately gauge the team's capacity, thereby bringing predictability to deliveries;"
        ],
        "biggestChallenge": "To manage a large team (28 professionals) and oversee critical deliveries."
      },
      "softplan-2": {
        "role": "Tech Lead",
        "description": "I was the technical leader of the Front-End chapter.",
        "accomplishments": [
          "Developed the new \"SAJ\" system, constructed using React JS, .Net, and GraphQL;",
          "Oversaw the deliveries and deployment of 17 modules of the system;",
          "Provided training in HTML, CSS, JavaScript, React JS, GraphQL, and unit tests;",
          "Led a team of 10 Front-End React developers;",
          "Recruited new employees for the company and new contractor professionals;"
        ],
        "biggestChallenge": "To acquire leadership skills and master people management."
      },
      "softplan-1": {
        "role": "Senior Front-End Developer",
        "description": "I was a Senior Front-End developer on the \"França\" team.",
        "accomplishments": [
          "Delivered the most critical and complex project in less than 6 months",
          "Developed new features for the \"e-SAJ\" portal utilizing React JS and GraphQL;",
          "Created new features for the \"SAJ Tribunais\" system using React JS and GraphQL;"
        ],
        "biggestChallenge": "To manage all the deliveries of the project as the sole Front-End developer."
      },
      "hotmart-3": {
        "role": "Senior Full-Stack Developer",
        "description": "I was a Senior Full-Stack developer on the \"Growth Hacking\" team.",
        "accomplishments": [
          "Developed systems to address the company's internal needs and automate tasks, leveraging technologies such as Vue.js, and Node.js bringing more than 1000 new leads;",
          "Created a Proof of Concept (POC) that introduced a simplified method for product registration on the platform;"
        ],
        "biggestChallenge": "To balance the need to comprehend customer requirements clearly while simultaneously coding."
      },
      "hotmart-2": {
        "role": "Mid Level Front-End Developer",
        "description": "I was a Mid Level Front-End developer on the \"User Acquisition\" team.",
        "accomplishments": [
          "Designed a system to generate blog post images using components and pre-defined templates, thereby enhancing the productivity of the design team by 1000%;",
          "Developed a high-performance caching system for the \"Hotmart blog\" (WordPress), which resulted in a 300% increase in performance;",
          "Implemented the Arabic version of the \"Hotmart blog\";"
        ],
        "biggestChallenge": "To conceptualize and develop complex solutions from the ground up."
      },
      "hotmart-1": {
        "role": "Entry Level Front-End Developer",
        "description": "I was an Entry Level Front-End developer on the \"User Acquisition\" team.",
        "accomplishments": [
          "Developed a new WordPress theme for the \"Hotmart blog\" utilizing only HTML5, CSS Grid Layout, and Vanilla JS;",
          "Implemented the new layout for the \"Hotmart blog\";",
          "Created custom features within the WordPress admin interface;"
        ],
        "biggestChallenge": "To acquire new development skills and apply them to complex projects."
      },
      "sydle-2": {
        "role": "Support Analyst",
        "description": "I was a Support Analyst on the \"Suporte & Sustentação\" team.",
        "accomplishments": [
          "Spearheaded the development of web projects utilizing WordPress and Magento;",
          "Facilitated the modeling and automation of the company's internal processes through BPMN;",
          "Provided comprehensive technical support to users of the company's systems;",
          "Oversaw the configuration and monitoring of both Linux and Windows servers."
        ],
        "biggestChallenge": "To rapidly acquire development skills and contribute to legacy projects."
      },
      "sydle-1": {
        "role": "Support Intern",
        "description": "I was a Support Intern on the \"Suporte & Sustentação\" team.",
        "accomplishments": [
          "Provided technical support to users of the company's systems;",
          "Configured and monitored infrastructure on Linux and Windows servers;",
          "Compiled team reports;",
          "Offered infrastructure support to the company's employees."
        ],
        "biggestChallenge": "To simultaneously learn a multitude of new concepts and balance them with my university commitments."
      }
    },
    "previousItemsText": {
      "prefix": "... and",
      "sufix": "other previous roles"
    },
    "imageAltText": {
      "prefix": "Logo of ",
      "sufix": " company"
    }
  },
  "education": {
    "title": "Education",
    "items": {
      "estacio": {
        "fieldOfStudy": "Strategic People Management",
        "degree": "Master of Business Administration - MBA",
        "subjects": [
          "Coaching and Leadership",
          "Communication in Organizations",
          "Conflict Management and Negotiation",
          "Corporative Education",
          "Creativity and Organizational Innovation",
          "Finance Principles",
          "Knowledge Management",
          "Strategic Vision in People Management",
          "Theory of Organizational Development"
        ]
      },
      "fumec": {
        "fieldOfStudy": "Computer Science",
        "degree": "Bachelor",
        "subjects": [
          "Analytical Algebra and Geometry",
          "Programming Language and Techniques I",
          "Computer Architecture",
          "Fundamentals of Distributed Systems",
          "Calculus I",
          "Introduction to Logical Systems",
          "Programming Language and Techniques II",
          "Computer Networks",
          "Data Structures I",
          "Calculus II",
          "Programming Language and Techniques III",
          "Data Structures II",
          "Introduction to Robotics",
          "Statistics",
          "Programming Language and Techniques IV",
          "Human Resources Administration",
          "Connectivity",
          "Algorithm Analysis",
          "Operating Systems and Basic Software",
          "Introduction to Administration",
          "Fundamentals of Physics",
          "Human-Machine Interface",
          "Theoretical Foundations of Computing",
          "Telecommunications",
          "Analysis and Systems Design",
          "Software Engineering I",
          "Database I",
          "Compilers",
          "Computer Graphics",
          "Software Engineering II",
          "Law Applied to Informatics",
          "Database II",
          "Numerical Calculus",
          "Information Systems Administration",
          "Information Technology Project Management",
          "Security in Computer Networks",
          "Final Course Work",
          "Artificial Intelligence",
          "Web Development",
          "Development of Distributed Systems"
        ]
      }
    },
    "imageAltText": {
      "prefix": "Logo of ",
      "sufix": " school"
    }
  },
  "skills": {
    "title": "Skills",
    "menu": {
      "all": "all",
      "hard": "hard",
      "soft": "soft"
    },
    "soft": {
      "communication": "Communication",
      "leadership": "Leadership",
      "problem-solving": "Problem Solving",
      "decision-making": "Decision Making",
      "empathy": "Empathy",
      "conflict-resolution": "Conflict Resolution",
      "adaptability": "Adaptability",
      "time-management": "Time Management",
      "organization": "Organization",
      "strategic-thinking": "Strategic Thinking",
      "mentoring": "Mentoring",
      "negotiation": "Negotiation",
      "resilience": "Resilience",
      "critical-thinking": "Critical Thinking",
      "creativity": "Creativity",
      "active-listening": "Active Listening",
      "team-building": "Team Building",
      "ethical-decision-making": "Ethical Decision Making",
      "customer-orientation": "Customer Orientation",
      "learning-agility": "Learning Agility",
      "curiosity": "Curiosity",
      "patience": "Patience",
      "public-speaking": "Public Speaking",
      "networking": "Networking",
      "multitasking": "Multitasking",
      "attention-to-detail": "Attention to Detail",
      "motivation": "Motivation",
      "persuasion": "Persuasion",
      "emotional-intelligence": "Emotional Intelligence"
    },
    "bottomText": "And always learning new things to expand the list..."
  },
  "recommendations": {
    "title": "Recommendations",
    "items": {
      "1665746687624": {
        "role": "Design Manager",
        "paragraphs": [
          "I had the opportunity to work with Daniel in co-leading the Design System at OLX, and his role was essential in building a strong community and an engaged team with the mission of scaling development and elevating the quality of our interfaces.",
          "He consistently made significant contributions in meetings, actively sharing his knowledge and insights to propel discussions forward. Furthermore, Daniel interacts effectively with all disciplines involved and identifies moments to mobilize teams towards impactful action.",
          "I highly recommend Daniel for leadership positions and reinforce my admiration for his professional commitment to business objectives and how he does not hesitate to invest effort in solving challenges along the way."
        ]
      },
      "1711736735188": {
        "role": "Senior Product Manager",
        "paragraphs": [
          "Daniel was a true partner specially when we were facing a business impact due to the pandemic. He worked non stop to come up with a solid business/tech plan together in order to recover from this scenario, also communicating what was needed to the management team.",
          "He is a people focused leader and makes effort to build a good team atmosphere. Also knows a lot about front-end technology and managed many different context teams."
        ]
      },
      "1692061488021": {
        "role": "Senior Staff Software Engineer",
        "paragraphs": [
          "I've had the opportunity to closely interact with Daniel during our teams' collaborative efforts, and I can attest to his qualities as a leader. Daniel brings positive energy to the table and demonstrates genuine interest in developing his team members and pairs. His commitment to delivering on agreed-upon solutions is unwavering, reflecting his dedication to meeting business objectives effectively. I recommend Daniel for any leadership role where his expertise will make a significant impact."
        ]
      },
      "1695594745530": {
        "role": "Product Designer",
        "paragraphs": [
          "Daniel is a very committed and dedicated professional. He is always interested in contributing in the best way possible and helping the team achieve its goals. If he doesn't know about the subject, he will try to collaborate in some way.",
          "He demonstrates a medium/long-term concern in his decisions, and manages to balance theory well with practice in his deliveries, which is essential for the healthy development of products and the team. In addition to having a great critical sense in several aspects, he clearly positively influenced the team's deliveries and strategic directions."
        ]
      },
      "1565735422940": {
        "role": "Principal Software Engineer",
        "paragraphs": [
          "Daniel is a great professional, always very committed. I worked with him at OLX Brasil, and saw him lead teams, always getting to know each of his team members, and using their skills positively to achieve relevant results. He was an example to be followed for his leadership, and will certainly be an excellent professional in his next challenges."
        ]
      },
      "1593552193665": {
        "role": "Software Engineering Manager",
        "paragraphs": [
          "Daniel is a very proactive and engaged professional. He was responsible for a more data-driven approach to monitoring web vitals at OLX Brasil, bringing greater visibility to the current situation and bringing the entire company to the topic. He has a strong problem-solving profile, strong leadership and achieves high engagement in his teams. Despite his management position, he has in-depth knowledge of front-end development, which helps with the technical development of his team members."
        ]
      },
      "1660079652443": {
        "role": "Senior Product Manager",
        "paragraphs": [
          "Daniel is an excellent, up-to-date professional. He demonstrates commitment, high technical knowledge, extremely collaborative. He always works for the common good and goals of the team and company. Always willing to share knowledge. Excellent addition to any company."
        ]
      },
      "1622852709278": {
        "role": "Engineering Manager",
        "paragraphs": [
          "Daniel is a very constructive person, who enjoys getting involved in different areas with assertive contributions and a lot of energy. He enjoys big challenges and being recognized for his contributions, sparing no effort to achieve goals. He also enjoys learning, relearning, collaborating, and sharing, enhancing other talents within the company. His good humor comes along with criticism regarding what can and should be improved, doing his part, seeking feedback, and asking for help with matters beyond his scope. Technically, he stood out in frontend work when we worked together, helping to shape other professionals who added a lot, and as a coordinator, he showed great organization, planning, and performance in complex projects, managing and caring for people, and providing good leadership within the team."
        ]
      },
      "1606086729239": {
        "role": "Data Analyst II",
        "paragraphs": [
          "Daniel is an excellent professional and I'd like to highlight his performance as a leader...",
          "I had the opportunity to work with him in two different teams and contexts. During this oppotunities I could witness a leader highly engaged in the development of each team member, including those ones that were not directly managed by him, such as me. Even during the period we were not on the same team, we had recurrent meetings, where he gave me a lot of orientations and helped me with my development.",
          "Furthermore, he was always concerned and showed interest in develop multidisciplinary solutions, bringing people from different areas of knowledge to the discussions.",
          "Daniel is a leader who is always available to help, guide and lead the team from the most everyday situations to the most urgent challenges."
        ]
      },
      "1683719518610": {
        "role": "Software Engineer",
        "paragraphs": [
          "During the period in which I was led by Dani I was able to witness his excellence as a manager. Your transparency stood out, transmitting confidence to the entire team!",
          "He is constantly concerned with the development of each team member, not only in terms of results, but also in career progression. He encouraged me to study relevant technical points and choose the tasks most beneficial to my growth and visibility within the company.",
          "Despite taking on a leadership role, Dani did not hesitate to get involved in technical issues. Actively collaborating on initiatives such as simplifying functions, solving problems with project components on other pages, and has always demonstrated a technical interest in understanding the functioning of different areas of the project to make the best decisions."
        ]
      },
      "1712709978402": {
        "role": "Engineering Manager",
        "paragraphs": [
          "I worked with Daniel and had the opportunity to be his leader and partner. As IC he was always very kind, proactive and very knowledgeable in frontend such as (JavaScript, React, Angular, etc.).",
          "As a peer, he demonstrated strong leadership skills in creating and managing teams. Excellent professional."
        ]
      },
      "1695221396520": {
        "role": "Software Engineer",
        "paragraphs": [
          "I had the pleasure of being led by Daniel at OLX. He was willing to help my professional growth from the beginning of our journey together, he was always available to help solve more complex problems and inspired me in the way he led with accurate questions and precise directions so that problems could be solved. This is because Daniel is a highly skilled professional in frontend stacks, with extensive knowledge of JavaScript. This knowledge, combined with the extensive experience he has as a leader and the calm and positive attitude he demonstrates, resulted in a very good experience on a daily basis. He was the best leader I ever had."
        ]
      },
      "1700516452017": {
        "role": "Senior Software Engineer",
        "paragraphs": [
          "I had the pleasure of working closely with Daniel Xavier at Softplan, where he excelled as our Tech Lead. Daniel is a standout front-end developer with expertise in JavaScript, React, TypeScript, and modern front-end practices like React Testing Library and GraphQL. His ability to architect scalable solutions and lead teams effectively sets him apart. Daniel's professionalism, reliability, and dedication to learning make him an invaluable asset to any team. I highly recommend Daniel for roles requiring top-tier front-end skills and strong leadership."
        ]
      },
      "1713311445833": {
        "role": "Senior Front-End Engineer",
        "paragraphs": [
          "Daniel is absolutely one of the most promising professionals I've met in my career!",
          "Very talkative professional, clear, and always have a point to put on the table to get rich some discussion.",
          "As tech lead (where I've worked with him), shows always open to listen and to help me and other colleagues.",
          "Daniel has the skill to see not only the micro but also the macro of the situation, so in this way supporting the team to achieve their goals in the best way possible!"
        ]
      }
    },
    "imageAltText": "Photo of the author"
  },
  "resume": {
    "title": "Resume",
    "sectionTitles": {
      "summary": "Summary",
      "skills": "Skills",
      "experience": "Experience",
      "education": "Education"
    },
    "biggestChallengeTitle": "Biggest challenge"
  },
  "not-found": {
    "text": "There is no page here.",
    "link": "Let's see a real page!",
    "imageAltText": {
      "prefix": "Photo of the author",
      "sufix": "indicating a link with his finger"
    }
  }
}
