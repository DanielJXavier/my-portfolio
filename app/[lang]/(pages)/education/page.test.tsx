import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import en from "dictionaries/en.json";

import Page, { generateMetadata } from "./page";

describe("Education page", () => {
  it("renders the page", () => {
    const { container } = render(
      <Page
        params={{
          lang: "en",
        }}
      />
    );

    expect(container).toMatchSnapshot();
  });

  it("generate the page metadata", async () => {
    const metadata = await generateMetadata({
      params: {
        lang: "en",
      },
    });

    const {
      education: { title },
    } = en;

    expect(metadata.title).toEqual(title);
  });
});
