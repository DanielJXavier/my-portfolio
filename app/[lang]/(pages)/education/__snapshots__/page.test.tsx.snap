// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Education page renders the page 1`] = `
<div>
  <h1
    class="text-xl md:text-2xl lg:text-3xl flex items-center gap-x-2"
    data-testid="title"
  >
    <svg
      class="fill-primary size-6 md:size-8"
      viewBox="0 -960 960 960"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="m479.38-166.16-267.3-146.3v-216.93L81.54-600l397.84-216.92L878.46-600v285.46h-45.39v-258.92l-86.38 44.07v216.93l-267.31 146.3Zm0-268.38L782.85-600 479.38-763.23 177.15-600l302.23 165.46Zm0 216.92 221.93-122v-162.23l-221.93 118.7-221.92-120.31v163.84l221.92 122Zm.62-216.92Zm-.62 60.54Zm0 0Z"
      />
    </svg>
    Education
  </h1>
  <article
    class="my-6 sm:my-10 relative before:content-[''] before:absolute before:top-[40px] before:bottom-[-21px] before:sm:bottom-[-36px] last-of-type:before:bottom-0 before:left-[16px] before:ml-0.5 before:w-1 before:bg-gradient-to-b before:from-[#2f4ccc] before:via-[#2f4ccc] before:to-background before:-z-10"
    data-testid="education-item"
  >
    <header
      class="flex items-start gap-x-2"
    >
      <img
        alt="Logo of Estácio school"
        class="rounded-full"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Feducation%2Festacio.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimages%2Feducation%2Festacio.jpg&w=48&q=75 1x, /_next/image?url=%2Fimages%2Feducation%2Festacio.jpg&w=96&q=75 2x"
        style="color: transparent;"
        width="40"
      />
      <h2
        class="text-base sm:text-lg md:text-xl xl:text-2xl font-bold leading-tight"
      >
        Strategic People Management
         @ 
        Estácio
        <span
          class="text-xs md:text-sm"
        >
           (
          2020 ~ 2021
          )
        </span>
      </h2>
    </header>
    <section
      class="pl-14 lg:px-14"
    >
      <h3
        class="xl:text-xl"
      >
        Master of Business Administration - MBA
      </h3>
    </section>
    <section
      class="mt-4 pl-14 xl:px-14"
    >
      <ul
        class="pl-7 list-disc text-sm xl:text-base"
      >
        <li>
          Coaching and Leadership
        </li>
        <li>
          Communication in Organizations
        </li>
        <li>
          Conflict Management and Negotiation
        </li>
        <li>
          Corporative Education
        </li>
        <li>
          Creativity and Organizational Innovation
        </li>
        <li>
          Finance Principles
        </li>
        <li>
          Knowledge Management
        </li>
        <li>
          Strategic Vision in People Management
        </li>
        <li>
          Theory of Organizational Development
        </li>
      </ul>
    </section>
  </article>
  <article
    class="my-6 sm:my-10 relative before:content-[''] before:absolute before:top-[40px] before:bottom-[-21px] before:sm:bottom-[-36px] last-of-type:before:bottom-0 before:left-[16px] before:ml-0.5 before:w-1 before:bg-gradient-to-b before:from-[#316ba9] before:via-[#316ba9] before:to-background before:-z-10"
    data-testid="education-item"
  >
    <header
      class="flex items-start gap-x-2"
    >
      <img
        alt="Logo of Universidade FUMEC school"
        class="rounded-full"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Feducation%2Ffumec.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimages%2Feducation%2Ffumec.jpg&w=48&q=75 1x, /_next/image?url=%2Fimages%2Feducation%2Ffumec.jpg&w=96&q=75 2x"
        style="color: transparent;"
        width="40"
      />
      <h2
        class="text-base sm:text-lg md:text-xl xl:text-2xl font-bold leading-tight"
      >
        Computer Science
         @ 
        Universidade FUMEC
        <span
          class="text-xs md:text-sm"
        >
           (
          2010 ~ 2014
          )
        </span>
      </h2>
    </header>
    <section
      class="pl-14 lg:px-14"
    >
      <h3
        class="xl:text-xl"
      >
        Bachelor
      </h3>
    </section>
    <section
      class="mt-4 pl-14 xl:px-14"
    >
      <ul
        class="pl-7 list-disc text-sm xl:text-base"
      >
        <li>
          Analytical Algebra and Geometry
        </li>
        <li>
          Programming Language and Techniques I
        </li>
        <li>
          Computer Architecture
        </li>
        <li>
          Fundamentals of Distributed Systems
        </li>
        <li>
          Calculus I
        </li>
        <li>
          Introduction to Logical Systems
        </li>
        <li>
          Programming Language and Techniques II
        </li>
        <li>
          Computer Networks
        </li>
        <li>
          Data Structures I
        </li>
        <li>
          Calculus II
        </li>
        <li>
          Programming Language and Techniques III
        </li>
        <li>
          Data Structures II
        </li>
        <li>
          Introduction to Robotics
        </li>
        <li>
          Statistics
        </li>
        <li>
          Programming Language and Techniques IV
        </li>
        <li>
          Human Resources Administration
        </li>
        <li>
          Connectivity
        </li>
        <li>
          Algorithm Analysis
        </li>
        <li>
          Operating Systems and Basic Software
        </li>
        <li>
          Introduction to Administration
        </li>
        <li>
          Fundamentals of Physics
        </li>
        <li>
          Human-Machine Interface
        </li>
        <li>
          Theoretical Foundations of Computing
        </li>
        <li>
          Telecommunications
        </li>
        <li>
          Analysis and Systems Design
        </li>
        <li>
          Software Engineering I
        </li>
        <li>
          Database I
        </li>
        <li>
          Compilers
        </li>
        <li>
          Computer Graphics
        </li>
        <li>
          Software Engineering II
        </li>
        <li>
          Law Applied to Informatics
        </li>
        <li>
          Database II
        </li>
        <li>
          Numerical Calculus
        </li>
        <li>
          Information Systems Administration
        </li>
        <li>
          Information Technology Project Management
        </li>
        <li>
          Security in Computer Networks
        </li>
        <li>
          Final Course Work
        </li>
        <li>
          Artificial Intelligence
        </li>
        <li>
          Web Development
        </li>
        <li>
          Development of Distributed Systems
        </li>
      </ul>
    </section>
  </article>
</div>
`;
